src/sniffer/Sniffer.c

```diff
@@ -0,0 +1,98 @@
+
+#include "Sniffer.h"
+
+#include <pcap.h>
+#include <time.h>
+#include <string.h>
+
+#include "IOHandler.h"
+
+#define WRITE_BUFFER_SIZE 1024
+
+/* 4 bytes IP address */
+typedef struct ip_address
```

Comment:
Should we move these to a header file? I don't see it necessary for now, but if we for example create a complex parser and want to move it out from this module, we will need these defs

Reply:
Yes, I was leaving it here for now only. I expect to move them out of here when a proper parser is implemented.